package.json

```diff
@@ -12,74 +12,68 @@
       "name": "Rafal Chlodnicki (@rchl)"
     }
   ],
-  "files": [
-    "lib",
-    "types"
-  ],
   "publishConfig": {
     "access": "public"
   },
-  "main": "lib/module.js",
-  "types": "types/index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/module.mjs",
+      "require": "./dist/module.cjs"
+    }
+  },
+  "main": "./dist/module.cjs",
+  "types": "./dist/types.d.ts",
+  "files": [
+    "dist",
+    "types"
+  ],
   "scripts": {
+    "prepack": "nuxt-module-build",
+    "dev": "nuxi dev playground",
+    "dev:build": "nuxi build playground",
+    "dev:prepare": "nuxt-module-build --stub && nuxi prepare playground",
     "dev:fixture": "node ./node_modules/nuxt/bin/nuxt.js -c ./test/fixture/lazy/nuxt.config.js",
-    "dev:fixture:build": "node ./node_modules/nuxt/bin/nuxt.js build -c ./test/fixture/default/nuxt.config.js",
-    "dev:fixture:start": "node ./node_modules/nuxt/bin/nuxt.js start -c ./test/fixture/default/nuxt.config.js",
-    "dev:generate": "nuxt generate -c ./test/fixture/default/nuxt.config.js --force-build",
-    "lint": "eslint --ext .vue,.js,.ts lib test types",
+    "lint": "eslint --ext .vue,.js,.ts src test types",
     "lint:fixture": "eslint --ext .vue,.js --no-ignore 'test/fixture/*/.nuxt/sentry.*'",
     "release": "release-it",
-    "test:fixture": "jest --runInBand",
-    "test": "yarn lint && yarn test:fixture && yarn lint:fixture",
+    "test:fixture": "NODE_OPTIONS=--experimental-vm-modules jest --runInBand",
+    "test": "yarn lint && yarn prepack && yarn test:fixture && yarn lint:fixture",
     "coverage": "codecov"
```

Comment:
Running `prepack` depends on `dev:prepare` due to `"extends": "./playground/.nuxt/tsconfig.json"` in `tsconfig.json`.
Running `dev:prepare` depends on `prepack` since it requires module to be built first.
So it's quite a conundrum.

Also, why is `prepack` not called `build`?

Reply:
I agree build would have more meaning, I left it as prepack as that was how it was called in the module builder template.
And no dev:prepare doesn't depend on prepack

Reply:
> And no dev:prepare doesn't depend on prepack

I'm not sure if I was clear. I meant that there is a cyclic dependency between those two. One requires the other to be run first for itself to work. So this kinda impossible situation unless one creates the `./playground/.nuxt` first by some means (by removing dependency on the Sentry module from the playground, for example).

Reply:
Actually tried it again and `dev:prepare` does seem to complete successfully.
Maybe let's then add a `"prepare": "yarn dev:prepare"` script so that this runs automatically on installing dependencies and doesn't require an extra step.

Reply:
> let's then add a `"prepare": "yarn dev:prepare"`

Nice idea!

Reply:
Just sharing some (late) context, the Nuxt 3 module template uses the `prepack` script hook to be sure the module gets build before `npm publish` (since a build step is now _mandatory_ to build modules when it wasn't with Nuxt 2 module flavor)